plugin/matrix.vim

```diff
@@ -301,7 +298,25 @@ function! s:Cleanup()
    let c = getchar(0)
 endfunction
 
-function! Matrix()
+function! s:ReadArgs(args)
```

Comment:
Try this instead (minor whitespace fix plus a range check):

```
function! s:ReadArgs(args)
   if len(a:args) == 0
      let [s:mindelay,s:maxdelay] = [1,5]
      return 0
   endif
   if len(a:args) == 2
      let [s:mindelay,s:maxdelay] = sort(map(copy(a:args), 'abs(str2nr(v:val))'), 'N')
      if s:mindelay > 0 && s:maxdelay > s:mindelay
         return 0
      endif
   endif
   return 1
endfunction
```

VIM actually handles divide by zero (see 'expr-%') but it just looks weird.